internal/config/notify/help.go

```diff
@@ -262,6 +262,18 @@ var (
 			Optional:    true,
 			Type:        "sentence",
 		},
+		config.HelpKV{
+			Key:         target.KafkaCompressionCodec,
+			Description: "specify compression_codec of the kafka",
```

Comment:
%s/specify compression_codec of the kafka/specify compression codec of the Kafka cluster/g